src/app/components/reports-page/reports-page.component.ts

```diff
@@ -1,8 +1,5 @@
 import { Component, OnInit } from '@angular/core';
-import { HttpClient } from '@angular/common/http';
```

Comment:
Be careful about removing imports in PRs that are not about cleaning/refactoring. This is okay for now, but in the future, even if they can be cleaned/removed, leave them be for a PR that is dedicated to cleaning/refactoring this type of thing.

Reply:
Understood. I will try not to remove them.